k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-svc.yaml

```diff
@@ -37,8 +37,8 @@ spec:
     - name: cortx-rgw-http
       protocol: TCP
       port: {{ .Values.cortxserver.service.loadbal.ports.http }}
```

Comment:
Are we missing the updates to the individual NodePort Services via `.Values.cortxserver.service.loadbal.ports.http` that reflect the same intent as the `cortx-io-svc` (configured via cortx-servers's values.yaml file)
https://github.com/Seagate/cortx-k8s/blob/b5ba60651caecfebcba672835a76b580a725b47c/k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/values.yaml#L22-L23

If I am reading everything else correctly, we'd want to update those defaults to 80 and 443 as well. I believe those are no longer needed from an architectural perspective, but we should keep them in sync with `cortx-io-svc` updates until we can remove them altogether.